run_linux.go

```diff
@@ -181,6 +181,17 @@ func (b *Builder) Run(command []string, options RunOptions) error {
 		}
 		bindFiles["/etc/resolv.conf"] = resolvFile
 	}
+	// Empty file, so no need to recreate if it exists
+	if _, ok := bindFiles["/run/.containerenv"]; !ok {
+		// Empty string for now, but we may consider populating this later
+		containerenvPath := filepath.Join(path, "/run.containerenv")
```

Comment:
should this be `/run/.containerenv`?  (added a slash after run)

Reply:
I love typos...  I guess tests would have found this.  Fixed.